crypto/fipsmodule/rand/cpu_jitter_test.cc

```diff
@@ -42,10 +42,6 @@ TEST(CPUJitterEntropyTest, Basic) {
   EXPECT_EQ(jent_read_entropy(jitter_ec.instance,
                               (char*) data0, data_len), data_len);
 
-  // Draw some entropy with the "safe" API to check if it works.
-  EXPECT_EQ(jent_read_entropy_safe(&jitter_ec.instance,
-                                   (char*) data1, data_len), data_len);
-
```

Comment:
I think the spirit of this test was a very simple sanity test that jitter was returning unique output based on line 49. Instead of removing this just use the same `jent_read_entropy` as above.